deployment/roles/traffic-configurator/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
This file is duplicated in `roles/injector-configurator/tasks/traffic-configurator.yml`. Which one is the valid one?

Reply:
Thanks for spotting that. I have removed the file in the path 

> deployment/roles/injector-configurator/tasks/traffic-configurator.yml

and pushed another commit to the branch.